dlc_developer_config.toml

```diff
@@ -21,14 +21,19 @@ datetime_tag = true
 do_build = true
 
 [test]
-# On by default
+### On by default
 sanity_tests = true
 ecs_tests = true
 eks_tests = true
 ec2_tests = true
 
-# Off by default
-sagemaker_tests = false
+### Off by default
+sagemaker_tests = true
+  # Setting release_candidate_tests to `true` will disable SM tests and run RC tests in SM test jobs.
```

Comment:
can be `will disable SM remote tests`?

Reply:
sure, will add that

Reply:
went with @SergTogul suggestion and changed this around. will resolve this for now